game-to-flatpak

```diff
@@ -49,6 +52,15 @@ for k, v in ipairs(arg) do
 	end
 end
 
+if not options.repo_dir then
+    options.repo_dir = 'repo'
+elseif not options.keep_repo_dir then
+    -- Adding it to your local repo and also making a bundle should not delete
+    -- the whole local repo.
+    print('Repo directory defined, assuming --keep-repo.')
+    options.keep_repo_dir = true
```

Comment:
It might have been a good idea to set `options.repo_dir` to something so that later...